src/wiki/ventoy.md

```diff
@@ -0,0 +1,106 @@
+# Creating a Multi-OS Bootable Drive (with Ventoy)
+In this article, you will learn how to create a multi-OS bootable drive. 
+
+## Creating the Bootable Drive
+This section will teach you how to create a bootable drive for your needs.
+1. Download Ventoy [here](https://www.ventoy.net/en/download.html). This is the tool that we will be using to create a bootable drive.
+::: Warning
+2. Plug in a USB drive that has at least 8GB.
+3. Right click on the file that you have just downloaded and select extract. Then, enter the folder named `ventoy-1.0.xx` and open an application named `Ventoy2Disk.exe`. A UAC prompt will pop-up, click yes.
+4. Select a drive in the device selection section. This will be your bootable drive. If you can't see the drive you want to use, press the green button to refresh the list.
+In this area, you will see how to create your USB for being bootable.
```

Comment:
You marked Acto's comment as resolved but it appears that nothing has changed on line 11?